### PR TITLE
Docs: clarify distinction between single binary and microservices.

### PR DESCRIPTION
**Pull request by KMiller-Grafana**
As a step towards providing examples that are easier for introductory Loki users to use (single binary and a minimal configuration), this PR attempts to clarify and simplify the prose around running as a single binary vs. running as microservices.  I fixed a few formatting/spelling typos, since I was updating the files anyway.

I did not re-draw the first diagram in the Architecture section, but will in another PR.

### CODE REVIEW

File: docs/sources/installation/helm.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ---
 title: Helm
+weight: 20
 ---
 # Install Loki with Helm
 
+The Helm installation runs the Loki cluster in microservices mode.
```
Comment (sandeepsukhani) [inline docs/sources/installation/helm.md:7]: The instructions here use helm charm which runs Loki in single binary mode. There is a `loki-distributed` chart that runs Loki in microservices mode but it is not covered here. Let us update the comment for now saying it runs Loki in single binary mode and then we can update it when we add the instructions for `loki-distributed`.

**Review by sandeepsukhani — APPROVED** [review]
Added some minor suggestions but other than that it LGTM.

File: docs/sources/installation/docker.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
 ---
 title: Docker
+weight: 30
 ---
 # Install Loki with Docker or Docker Compose
 
 You can install Loki and Promtail with Docker or Docker Compose if you are evaluating, testing, or developing Loki.
 For production, we recommend installing with Tanka or Helm.
 
+The configuration acquired by download specifies running Loki as a single binary.
```
[the remark this answers is not in the document]
Comment (KMiller-Grafana) [reply docs/sources/installation/docker.md]: Thanks, @sandeepsukhani. I've reworded the sentence.

File: docs/sources/installation/helm.md (hunk 1 above)
Comment (KMiller-Grafana) [reply docs/sources/installation/helm.md]: Correction implemented.  Thanks.

**Review by owen-d — APPROVED** [review]

**Pull request merged**